lib/rdoc/generator/template/rails/resources/css/panel.css

```diff
@@ -12,6 +96,34 @@
         line-height: 1;
     }
 
+    @media (max-width: 39.99em) {
+        .panel
+        {
+            -webkit-transform:translate(-100%, 0);
+            -moz-transform:translate(-100%, 0);
+            -ms-transform:translate(-100%, 0);
+            transform:translate(-100%, 0);
+             -webkit-transition:transform 0s ease-in-out;
+            -moz-transition:transform 0s ease-in-out;
+            -ms-transition:transform 0s ease-in-out;
+            transition:transform 0s ease-in-out;
+            opacity: 0;
+            width: 100%
+        }
+
+        .panel_checkbox:checked ~ .panel {
+            -webkit-transform:translate(0, 0);
+            -moz-transform:translate(0, 0);
+            -ms-transform:translate(0, 0);
+            transform:translate(0, 0);
+            -webkit-transition:transform 0.3s ease-in-out;
+            -moz-transition:transform 0.3s ease-in-out;
+            -ms-transition:transform 0.3s ease-in-out;
+            transition:transform 0.3s ease-in-out;
```

Comment:
*Question:* I stopped using the vendor prefixes along with autoprefixer in most my projects on the thought that that specific feature is pretty standard now. Are they required?

Reply:
Fair point ; [Can I Use](https://caniuse.com/mdn-css_properties_transition) states that we would be compatible with 95% of the current market share with the unprefixed version. Given the nature of the audience, we can assume users have an up-to-date browser.

Reply:
Ok, I'll remove the prefixes. 👍